src/cli.rs

```diff
@@ -199,6 +199,14 @@ pub(crate) enum Subcommand {
     )]
     Clean(CleanOptions),
 
+    /// Run tests with cargo nextest
+    #[clap(
+        bin_name = "cargo llvm-cov nextest",
+        max_term_width(MAX_TERM_WIDTH),
+        setting(AppSettings::DeriveDisplayOrder)
+    )]
+    Nextest,
```

Comment:
Options after the `nextest` subcommand do not seem to be recognized.

```
$ cargo llvm-cov nextest --all-features
error: Found argument '--all-features' which wasn't expected, or isn't valid in this context

        If you tried to supply `--all-features` as a value rather than a flag, use `-- --all-features`

USAGE:
    cargo llvm-cov nextest

For more information try --help
```

Reply:
I believe https://github.com/facebookincubator/cargo-guppy/blob/main/tools/cargo-hakari/src/command.rs#L250 should work.

Reply:
That's awesome! I'll have a try :)